tests/integration/__tests__/debug-api.js

```diff
@@ -72,18 +72,6 @@ describe("API", () => {
       await formatDoc(fill(["foo", hardline, "bar", literalline, "baz"])),
     ).toBe('fill(["foo", hardline, "bar", literalline, "baz"])');
 
-    /*
-    This is not really `cleanDoc` from `src/document/utils.js`
-    But if we pass array to it, it will flat array
-    */
-    const cleanDoc = (parts) => parts.flat();
-    expect(
-      await formatDoc(
-        // The argument of fill must not be passed to cleanDoc because it's not a doc
-        fill(cleanDoc(["foo", literalline, "bar"])), // invalid fill
-      ),
-    ).toBe('fill(["foo", literallineWithoutBreakParent, breakParent, "bar"])');
-
```

Comment:
I'm considering this test case doesn't make sense. I'll add some tests if we need alternative tests.